test/test-debuglet.ts

```diff
@@ -377,6 +377,32 @@ describe('Debuglet', () => {
       assert.deepEqual(mergedConfig, compareConfig);
     });
 
+    it('should not start when workingDirectory is the a directory', (done) => {
+      const debug = new Debug({}, packageInfo);
+      const config = extend({}, defaultConfig, {workingDirectory: path.sep});
+      const debuglet = new Debuglet(debug, config);
+      let text = '';
+      debuglet.logger.error = (str: string) => {
+        text += str;
+      };
+
+      debuglet.on('initError', (err: Error) => {
+        assert.ok(err);
```

Comment:
Maybe use `assert.ifError` here? And three lines below use `assert.ok`

Reply:
`assert.ifError(err)` would cause the test to fail if `err` is defined.  Instead, I want to verify that `err` is defined and so I used `ok`.  I can switch the other `assert` to `assert.ok` since they are aliases if you think it is clearer.

Reply:
Updated `assert` to `assert.ok`.